BlockHedge/Pages/Home.razor

```diff
@@ -76,23 +77,39 @@
                 {
                     <div class="voting-overview">
                         <h3 class="h6 mb-2">Vote on Ideas</h3>
-                        <div class="current-proposal mb-3">
-                            <h4 class="h6 mb-2">Current Proposal</h4>
-                            <p class="proposal-description mb-2 small">@currentProposal.Description</p>
-                            <div class="voting-buttons">
-                                <button class="btn btn-success btn-sm me-2 mb-2 mb-md-0" @onclick="() => Vote(true)">Yes</button>
-                                <button class="btn btn-danger btn-sm" @onclick="() => Vote(false)">No</button>
-                            </div>
-                            <div class="voting-results mt-2">
-                                <div class="progress">
-                                    <div class="progress-bar bg-success" role="progressbar" style="width: @(currentProposal.YesPercentage)%" aria-valuenow="@currentProposal.YesPercentage" aria-valuemin="0" aria-valuemax="100">@currentProposal.YesPercentage%</div>
-                                    <div class="progress-bar bg-danger" role="progressbar" style="width: @(currentProposal.NoPercentage)%" aria-valuenow="@currentProposal.NoPercentage" aria-valuemin="0" aria-valuemax="100">@currentProposal.NoPercentage%</div>
+                        @if (proposals.Count > 0)
+                        {
+                            @for (int i = startIndex; i < Math.Min(startIndex + pageSize, proposals.Count); ++i)
+                            {
+                                var proposal = proposals[i];
+                                <div class="current-proposal mb-3">
+                                    <h4 class="h6 mb-2">Proposal @(i + 1): @proposal.Title</h4>
+                                    <p class="proposal-description mb-2 small">@proposal.Description</p>
+                                    <div class="voting-buttons">
+                                        <button class="btn btn-success btn-sm me-2 mb-2 mb-md-0" @onclick="() => Vote(true, i - 1)">Yes</button>
+                                        <button class="btn btn-danger btn-sm" @onclick="() => Vote(false, i - 1)">No</button>
```

Comment:
@coderabbitai For some reason it always started at 1 even though startindex was initialized to 0. So what I did is implement a quick dirty fix  (i-1).

Why do you think it always started at 1?

Reply:
@coderabbitai i - 1 works. But before, when I was using just "i", it started at 1. And I dont know why.

Reply:
@coderabbitai Actually, when the VoteYes or VoteNo function is triggered, it passes 1. 
The smartcontract then throws an error because proposal 1 doesnt exist. The smartcontract starts counting at 0, not 1.

There is something else going wrong.

Reply:
@coderabbitai
> You're passing proposalIndex directly to WalletService.VoteYes or WalletService.VoteNo:
> 
> private async Task Vote(bool voteYes, int proposalIndex)
> {
>     if (voteYes)
>     {
>         await WalletService.VoteYes(proposalIndex);
>     }
>     else
>     {
>         await WalletService.VoteNo(proposalIndex);
>     }
> }

Thats actually not the case. In the `Vote` function, (given that I dont use i - 1), proposalIndex is already 1 when it should be 0...

Reply:
@coderabbitai Please create a github issue for this. I will get to it another time day.